stable/oauth2-proxy/README.md

```diff
@@ -73,6 +73,9 @@ Parameter | Description | Default
 `resources` | pod resource requests & limits | `{}`
 `service.port` | port for the service | `80`
 `service.type` | type of service | `ClusterIP`
+`service.clusterIP` | cluster ip address | `nil`
```

Comment:
Is this a pattern used in other Charts?

Reply:
yea, took an example out of grafana https://github.com/helm/charts/blob/master/stable/grafana/templates/service.yaml#L18